### DISCUSSION TITLE
Is there a way to open the webcam from web mode

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#1209] [Q&A]
### Question

I want to use the webcam from flet . How to open a webcam and show it on the web . Or is there a way to run a javascript script on the flet so that the webcam opens

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by hololeo** [comment]
You can connect the two with a websocket proxy. Or you can use some apis that pyodide provides. Im working on a small project to assist with this and it should be out in a few weeks.

**Comment by monkeycc** [comment]
opencv RTSP

**Reply by ndonkoHenri** [reply]
Check this: https://github.com/azu2023/CODES/blob/main/cv_video.py

**Reply by bobwatcherx** [reply]
> Check this: https://github.com/azu2023/CODES/blob/main/cv_video.py

I want a video webcam on my website. and what does this video path mean for?

cap = cv2.VideoCapture("Your video path")

**Reply by ndonkoHenri** [reply]
Use this to open the webcam:

```python 
cap = cv2.VideoCapture(0)
```

Gotten from https://www.geeksforgeeks.org/python-opencv-capture-video-from-camera/amp/

**Comment by goyalyashpal** [comment]
* https://github.com/flet-dev/flet/issues/344
* :star: https://github.com/flet-dev/flet/issues/648
+ https://github.com/flet-dev/flet/discussions/1209
* https://github.com/flet-dev/flet/issues/1281
* https://github.com/flet-dev/flet/issues/1268
